.github/workflows/ci.yml

```diff
@@ -68,10 +68,11 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Build communication container
+        if: matrix.toxenv == 'py311' || matrix.toxenv == 'py312'
         working-directory: ./resolwe/flow/docker_images
         run: docker build -f Dockerfile.communication -t ${{ env.RESOLWE_COMMUNICATOR_IMAGE }} ../../
 
-      - name: Test
+      - name: ${{ matrix.toxenv }}
```

Comment:
Add a more descriptive name. It may contain `toxenv`, but it should also explain what the step does.